src/test/java/org/opentripplanner/graph_builder/module/GtfsModuleTest.java

```diff
@@ -73,18 +73,20 @@ static GtfsBundle bundle(String feedId) {
       return b;
     }
 
-    static Stream<Arguments> interliningCases = Stream.of(
-      Arguments.of(List.of(bundle("A")), 2),
-      Arguments.of(List.of(bundle("A"), bundle("B")), 4),
-      Arguments.of(List.of(bundle("A"), bundle("B"), bundle("C")), 6)
-    );
+    static Stream<Arguments> interliningCases() {
+      return Stream.of(
```

Comment:
```suggestion
      return List.of(
```

Probably ok without but safer if you switch to a List.

Reply:
You need to check if it works with MethodSource

Reply:
I think we would be ok with a stream but changed to List just to be sure.

Reply:
MethodSource works with any collection.